src/serializer/ResidualHeapSerializer.js

```diff
@@ -154,6 +162,10 @@ export class ResidualHeapSerializer {
   delayInitializations: boolean;
   referencedDeclaredValues: Set<AbstractValue>;
   activeGeneratorBodies: Map<Generator, Array<BabelNodeStatement>>;
+  additionalFunctionValuesAndEffects: Map<FunctionValue, Effects> | void;
+  // function values we aren't allowed to delay initializations in because they're
```

Comment:
Why are we not allowed to delay initializations in additional functions? Feature cut? Then add a TODO to reconsider this. Or is there a deeper reason why it would never work?